packages/application-config/src/schema.ts

```diff
@@ -26,14 +22,44 @@ export interface JSONSchemaForCustomApplicationConfigurationFiles {
    * The URL of the Merchant Center API. We usually recommend to use the `cloudIdentifier` option to avoid possible typos.
    */
   mcApiUrl?: string;
+  /**
+   * The permissions configuration for the required OAuth Scopes for this Custom Application.
+   */
+  permissions?: {
```

Comment:
Should we call this scopes? Just thinking out loud. Permissions infer scopes in the MC. This doesn't infer anything it's just scopes. Without the indirection of permissions.

Reply:
Initially I went with this because it's more or less what we wanted to show in the UI (registration form).
We then changed it and went with view/manage oauth scopes.

I'm fine remaining this, if you think it's confusing. Thanks for the reminder.

Reply:
I think it would help. The confusing bit could be otherwise that you might think that by adding "orders" here under "view" that I'd get the scopes the View Orders permission implies: e.g. types. That's not true tho.

Reply:
Changed it: https://github.com/commercetools/merchant-center-application-kit/pull/1934/commits/43934feb54e6fd7d2ae7ddbbf5cb60be557a5aec